api/src/main/java/org/openmrs/module/Module.java

```diff
@@ -19,6 +19,7 @@
 import java.util.Properties;
 import java.util.Set;
 import java.util.Vector;
+import java.lang.String;
```

Comment:
`java.lang.*` is imported by default. So there is no need to explicitly import `java.lang.String`